odpc.client/src/features/publicatie/use-documenten.ts

```diff
@@ -130,16 +130,6 @@ export const useDocumenten = (pubUUID: ComputedRef<string | undefined>) => {
 
   const removeDocument = (index: number) => documenten.value.splice(index, 1);
 
-  const toggleDocument = (uuid: string) => {
```

Comment:
is dit bewust verwijderd, of een merge foutje?

Reply:
Nee, bewust verwijderd, werd niet meer gebruikt.